experiments/yarn/src/main/resources/reference.conf

```diff
@@ -1,4 +1,4 @@
-gearpump {
+configuration {
```

Comment:
There is a tradition to put every config under path gearpump, so that we can easy filter out what configs we provides.

Would you mind put every config in this project under 

gearpump {
  yarn {
    //other configs
  }
}